webapp/tokens/cli.py

```diff
@@ -0,0 +1,39 @@
+import click
```

Comment:
So the philosophy you're going with here is that you keep all stuff to do with tokens in one folder, including CLI commands that are used in local development (apart from the Model which you've left in `webapp/models.py` which seems a little inconsistent).

The inverts the structure from what it was, which is that all mappers were in `webapp/mappers.py`, local development scripts in a `script` dir, etc.

I think which of these two opposing structures we choose should depend on what the need of the person working on (or reading) the project code is likely to be, and which aids understanding of how the app when you are trying to read it.

While I can see that when you're writing it, you might create all things to do with Tokens at the same time, I think when you're actually maintaining it you're much more likely to want to just work on one piece. E.g., most people in local development might be using the token CLI stuff to create local development tokens, but they won't otherwise case about tokens at all. Or they might be trying to find the code that processes the `/tokens` list page to see how that works.

In my opinion it's both a more common structure for a framework and also simpler to understand if we collect all views together in `webapp/views.py`, all models in `webapp/models.py` etc., and keep any CLI tools for local development in an entirely separate space from the code for the webapp. But I'm open to hearing opposing arguments - we just need to justify them from the point of view of the project maintainers or contributors.